scripts/darwin.sh

```diff
@@ -60,8 +60,11 @@ for disk in $DISKS; do
   else
     echo "mountpoints:"
     echo "$mountpoints" | while read -r mountpoint ; do
+      volume="$(echo "$mount_output" | perl -n -e'm{^('"${disk}"'(s[0-9]+)?) on (.*) \(.*\)$} && print "$1\n"')"
```

Comment:
Could this be modified to not use `perl` ?
We wouldn't want to see https://github.com/resin-io/etcher/issues/1288 getting re-introduced ;-)